avocado.spec

```diff
@@ -29,24 +29,50 @@ these days a framework) to perform automated testing.
 
 %prep
 %setup -q -n %{name}-%{commit}
+# package plugins-runner-vm requires libvirt-python, but the RPM
+# version of libvirt-python does not publish the egg info and this
+# causes that dep to be attempted to be installed by pip
+sed -e "s/'libvirt-python'//" -i optional_plugins/runner_vm/setup.py
```

Comment:
I agree.  I have to file a bug report on that:

https://trello.com/c/csNjIVJ6/920-file-bug-report-for-libvirt-python-package